courses/free-courses-en.md

```diff
@@ -439,6 +439,7 @@
 * [Express JS Crash Course](https://www.youtube.com/watch?v=L72fhGm1tfE) - Brad Traversy, Traversy Media (Youtube)
 * [Node.js API Development for Beginners](https://www.udemy.com/course/node-js-api-tutorial/) (Udemy)
 * [Node.js Basics](https://www.youtube.com/playlist?list=PLqq-6Pq4lTTa-d0iZg41U2RDqECol9C5B) - Java Brains (Youtube)
+* [Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) by [John Smilga](https://github.com/john-smilga) - Coding Addict (YouTube)
```

Comment:
Maybe this solve the linting errors

```
* [Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) - John Smilga, Coding Addict (YouTube)
```

Great tutorial @john-smilga

Reply:
@davorpa I have fixed that error and organised the list properly.

Reply:
is there no playlist? is it just one video?

Reply:
@eshellman No, actually this is an 8-hour long video covering all the aspects. For ease of navigation, they have provided topic-wise timestamps in the video description.

Reply:
Maybe the playlist will be this: https://www.youtube.com/playlist?list=PLnHJACx3NwAdT_8forzXYvx0o4A2VnoHX

But this is only have this video

Reply:
Exactly @davorpa , that playlist contains 10 different videos from 10 different topics 😃

Reply:
> Exactly @davorpa , that playlist contains 10 different videos from 10 different topics 😃

I see the video at a glance and for me it's fine. It covers all things. Chapters are attached as timeline blocks as @basuabhirup said.

Eric has the last word 😉

Reply:
most YT videos are not, by themselves long enough to qualify as a course. This one certainly is. (8 hours!!!) Probably should add something in the contrib guidelines. 
Thanks, and welcome to free-programming-books!